code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -74,56 +85,86 @@
 				second_sound_played = TRUE
-			icon_state = "clockwork_gateway_active"
-		if(250 to 300)
+			make_glow()
+			glow.icon_state = "clockwork_gateway_active"
+		if(GATEWAY_RATVAR_COMING to GATEWAY_RATVAR_ARRIVAL)
 			if(!third_sound_played)
 				world << sound('sound/effects/clockcult_gateway_closing.ogg', 1, channel = 8, volume = 50)
 				third_sound_played = TRUE
-			icon_state = "clockwork_gateway_closing"
-		if(300 to INFINITY)
+			make_glow()
+			glow.icon_state = "clockwork_gateway_closing"
+		if(GATEWAY_RATVAR_ARRIVAL to INFINITY)
 			if(!purpose_fulfilled)
 				takes_damage = FALSE
 				purpose_fulfilled = TRUE
-				animate(src, transform = matrix() * 1.5, alpha = 255, time = 126)
+				make_glow()
+				animate(glow, transform = matrix() * 1.5, alpha = 255, time = 126)
 				world << sound('sound/effects/ratvar_rises.ogg', 0, channel = 8) //End the sounds
 				sleep(131)
-				animate(src, transform = matrix() * 3, alpha = 0, time = 5)
+				make_glow()
+				animate(glow, transform = matrix() * 3, alpha = 0, time = 5)
 				sleep(5)
 				new/obj/structure/clockwork/massive/ratvar(get_turf(src))
 				qdel(src)
 
 /obj/structure/clockwork/massive/celestial_gateway/examine(mob/user)
+	icon_state = "spatial_gateway" //cheat wildly by pretending to have an icon
 	..()
+	icon_state = initial(icon_state)
 	if(is_servant_of_ratvar(user))
 		var/arrival_text = "IMMINENT"
-		if(300 - progress_in_seconds > 0)
-			arrival_text = "[max((300 - progress_in_seconds) / GATEWAY_SUMMON_RATE, 0)]"
+		if(GATEWAY_RATVAR_ARRIVAL - progress_in_seconds > 0)
+			arrival_text = "[round(max((GATEWAY_RATVAR_ARRIVAL - progress_in_seconds) / (GATEWAY_SUMMON_RATE * 0.5), 0), 1)]"
 		user << "<span class='big'><b>Seconds until Ratvar's arrival:</b> [arrival_text]s</span>"
 		switch(progress_in_seconds)
-			if(-INFINITY to 100)
+			if(-INFINITY to GATEWAY_REEBE_FOUND)
 				user << "<span class='heavy_brass'>It's still opening.</span>"
-			if(100 to 250)
+			if(GATEWAY_REEBE_FOUND to GATEWAY_RATVAR_COMING)
 				user << "<span class='heavy_brass'>It's reached the Celestial Derelict and is drawing power from it.</span>"
-			if(250 to INFINITY)
+			if(GATEWAY_RATVAR_COMING to INFINITY)
 				user << "<span class='heavy_brass'>Ratvar is coming through the gateway!</span>"
 	else
 		switch(progress_in_seconds)
-			if(-INFINITY to 100)
+			if(-INFINITY to GATEWAY_REEBE_FOUND)
 				user << "<span class='warning'>It's a swirling mass of blackness.</span>"
-			if(100 to 250)
+			if(GATEWAY_REEBE_FOUND to GATEWAY_RATVAR_COMING)
 				user << "<span class='warning'>It seems to be leading somewhere.</span>"
-			if(250 to INFINITY)
+			if(GATEWAY_RATVAR_COMING to INFINITY)
 				user << "<span class='warning'><b>Something is coming through!</b></span>"
 
+/obj/effect/clockwork/gateway_glow //the actual appearance of the Gateway to the Celestial Derelict; an object so the edges of the gate can be clicked through.
```

Comment:
It can't just be an overlay, oh nooooo. Overlays don't let you set mouse_opacity to anything other than the overlayed object's mouse_opacity.